packages/migrations/src/actions/2025.01.02T00-00-00.cascade-deletion-indices.ts

```diff
@@ -0,0 +1,127 @@
+import { type MigrationExecutor } from '../pg-migrator';
+
+/**
+ * This migration adds a bunch of indices for columns that have cascade triggers.
+ * Adding these significantly improves the delete performance.
```

Comment:
thank you! I think this should also solve the slowness around `deleteProject`? I noticed it's sometime slow

Reply:
It should improve the performance of that, as the cascade deletes around schema checks and schema publishes were the main culprits.